upgrade/Stage_0_Prerequisites.md

```diff
@@ -186,57 +227,254 @@ The http proxy variables must be `unset` after the desired artifacts are downloa
    git push
    ```
```

Comment:
It would be nice to have a note at the end of stage 0.2 to return to IUF documentation when performing a whole stack upgrade (i.e. CSM + all products)